jenkins-master/jobs/mozilla-aurora_functional/config.xml

```diff
@@ -48,32 +48,9 @@
       <contributors/>
     </EnvInjectJobProperty>
   </properties>
-  <scm class="hudson.plugins.git.GitSCM" plugin="git@2.3.5">
-    <configVersion>2</configVersion>
-    <userRemoteConfigs>
-      <hudson.plugins.git.UserRemoteConfig>
-        <url>https://github.com/mozilla/firefox-ui-tests.git</url>
-      </hudson.plugins.git.UserRemoteConfig>
-    </userRemoteConfigs>
-    <branches>
-      <hudson.plugins.git.BranchSpec>
-        <name>*/mozilla-aurora</name>
-      </hudson.plugins.git.BranchSpec>
-    </branches>
-    <doGenerateSubmoduleConfigurations>false</doGenerateSubmoduleConfigurations>
-    <submoduleCfg class="list"/>
-    <extensions>
-      <hudson.plugins.git.extensions.impl.LocalBranch>
-        <localBranch>mozilla-aurora</localBranch>
-      </hudson.plugins.git.extensions.impl.LocalBranch>
-      <hudson.plugins.git.extensions.impl.CloneOption>
-        <shallow>true</shallow>
-        <reference></reference>
-      </hudson.plugins.git.extensions.impl.CloneOption>
-      <hudson.plugins.git.extensions.impl.RelativeTargetDirectory>
-        <relativeTargetDir>firefox-ui-tests</relativeTargetDir>
-      </hudson.plugins.git.extensions.impl.RelativeTargetDirectory>
-    </extensions>
```

Comment:
If I recall correctly, gittool doesn't support it.

Maja Frydrychowicz
irc: maja_zf

On Thu, Nov 19, 2015 at 4:36 PM, Henrik Skupin <notifications@github.com>
wrote:

> In jenkins-master/jobs/mozilla-aurora_functional/config.xml
> <https://github.com/mozilla/mozmill-ci/pull/698#discussion_r45403966>:
>
> > -      </hudson.plugins.git.BranchSpec>
> > -    </branches>
> > -    <doGenerateSubmoduleConfigurations>false</doGenerateSubmoduleConfigurations>
> > -    <submoduleCfg class="list"/>
> > -    <extensions>
> > -      <hudson.plugins.git.extensions.impl.LocalBranch>
> > -        <localBranch>mozilla-aurora</localBranch>
> > -      </hudson.plugins.git.extensions.impl.LocalBranch>
> > -      <hudson.plugins.git.extensions.impl.CloneOption>
> > -        <shallow>true</shallow>
> > -        <reference></reference>
> > -      </hudson.plugins.git.extensions.impl.CloneOption>
> > -      <hudson.plugins.git.extensions.impl.RelativeTargetDirectory>
> > -        <relativeTargetDir>firefox-ui-tests</relativeTargetDir>
> > -      </hudson.plugins.git.extensions.impl.RelativeTargetDirectory>
> > -    </extensions>
>
> Why is the revision of the firefox-ui-tests repository not taken from the
> firefox-media-tests repository? Usually you use git submodule init/update
> but maybe gittool doesn't support it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/mozmill-ci/pull/698/files#r45403966>.
>